### DISCUSSION TITLE
FR: ability to set the default widgets per user group

### DISCUSSION THREAD

**Discussion by noregt** [craftcms/cms#8536] [Ideas]
We should be able to set the default widgets for a user group. Right now everybody gets the craft news widget and recent posts. For the really basic users I prefer no widgets at all for instance.

**Comment by RichardJong** [comment]
I managed to do this by extending the Craft Dashboard service, overriding the _addDefaultUserWidgets function. Would be helpful to set this in the CMS though.

**Comment by ryanmasuga** [comment]
I'd like this ability, too. For example, I don't know a single user from any entity for whom we've built a site that would care about Craft news.

**Comment by brandonkelly** [comment]
Sick burn, Ryan

**Comment by ryanmasuga** [comment]
? 

I'm dead serious. I just got off a call with a client less then 30 minutes ago who asked how to get rid of the "news block" and how can they add links that they all can see - which is why I came to find this Github issue in the first place.  

Didn't mean to sound like I'm burning anyone. It's just noise they have to look past every time they log in and have to try to remember how to do what they logged in to do. I'd like a clean set of default widgets that can be more relevant to those logging in.
